CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 [Full Changelog](https://github.com/stargate/data-api/compare/v1.0.10...v1.0.11)
 
+**Highlights:**
+
+- Support includeSortVector option for find and findOne command (API addition)
+- Improvements to vectorize feature
+  - Adds HuggingFace dedicated provider support
```

Comment:
Not 100% sure but I thought MarkDown requires 3 or more spaces for indentation for lists to work? If 2 is enough it's fine.
(was trying to look at f.ex https://stackoverflow.com/questions/53866598/markdown-list-indentation-with-3-spaces-or-4-spaces-what-is-the-standard which suggests 3 but not sure still)

Reply:
Looked at the "Rich diff" and it seems ok

![Screenshot 2024-06-18 at 1 32 38 PM](https://github.com/stargate/data-api/assets/12115970/ec40d7a7-2d20-458f-bbbb-c81f4a3405f7)